src/pgrn-list-broken-indexes.c

```diff
@@ -62,13 +137,19 @@ pgroonga_list_broken_indexes(PG_FUNCTION_ARGS)
 			continue;
 		}
 
+		if (!PGrnIsLockedSources(index) && !PGrnIsLockedLexicon(index))
```

Comment:
How about adding `PGrnIndexIsBroken()` and use it here?

```suggestion
		if (!PGrnIndexIsBroken(index))
```